### DISCUSSION TITLE
[Discussion] - AI tech, and the future of MAS

### DISCUSSION THREAD

**Discussion by WingdingsMaster** [Monika-After-Story/MonikaModDev#7914] [general]
just some ideas I had, there is a website I found [This](https://www.cnet.com/how-to/samsung-neon-artificial-humans-are-confusing-everyone-we-set-record-straight/) and it got me thinking, "Is this the possible future of MAS?" just saying, it could be, sometime in the future

**Comment by Rai99** [comment]
It's definitely possible, but probably not for a while, especially without proper backing.
